examples/energy-management-app/energy-management-common/include/EnergyManagementAppCmdLineOptions.h

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <lib/support/BitMask.h>
+
+chip::BitMask<chip::app::Clusters::DeviceEnergyManagement::Feature> GetFeatureMap();
```

Comment:
`GetFeatureMap` without an namespace at a top level seems too generic.

Reply:
Have put in a namespace and renamed function to GetFeatureMapFromCmdLine